packages/app/src/containers/FillInspector.tsx

```diff
@@ -9,7 +9,13 @@ import {
 } from '../contexts/ApplicationStateContext';
 import useShallowArray from '../hooks/useShallowArray';
 
-export default memo(function FillInspector() {
+export default memo(function FillInspector({
+  title = 'Fill',
```

Comment:
Let's remove the default values and pass them into both `<FillInspector />`s. I think it'll be clearer to be explicit here

Reply:
okey!